src/gl_fns.rs

```diff
@@ -1123,6 +1123,14 @@ impl Gl for GlFns {
         }
     }
 
+    fn get_vertex_attrib_pointer(&self, index: GLuint, pname: GLenum) -> GLsizeiptr {
```

Comment:
shouldn't it have `_v` suffix, similar to how `get_vertex_attrib*` have suffixes?